erpnext/stock/doctype/material_request/material_request.py

```diff
@@ -397,15 +398,17 @@ def raise_production_orders(material_request):
 	production_orders = []
 	default_wip_warehouse = frappe.db.get_single_value("Manufacturing Settings", "default_wip_warehouse")
 	for d in mr.items:
-		if (d.qty - d.ordered_qty) >0:
+		qty = d.qty - (d.ordered_qty / d.conversion_factor)
```

Comment:
Production Order does not have multi uom / conversion factor. You only need to fix `if (d.stock_qty - d.ordered_qty) >0:`